### PR TITLE
Add Workflow Run show page

### PR DESCRIPTION
**Pull request by Devessier**
In this PR:

- Display a workflow version visualizer for the version of the workflow the run was executed on.
- Display the output of the run as code.

https://github.com/user-attachments/assets/d617300a-bff4-4328-a35c-291dc86d81cf

### CODE REVIEW

File: packages/twenty-front/src/modules/workflow/components/WorkflowRunOutputVisualizer.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+}: {
+  workflowRunId: string;
+}) => {
+  const workflowRun = useWorkflowRun({ workflowRunId });
+  if (!isDefined(workflowRun)) {
+    return null;
+  }
+
+  return (
+    <StyledSourceCode>
```
Comment (martmull) [inline packages/twenty-front/src/modules/workflow/components/WorkflowRunOutputVisualizer.tsx:25]: I think you could use the <CodeEditor component here. See an example in readonly CodeEditor usage in `packages/twenty-front/src/modules/settings/serverless-functions/components/tabs/SettingsServerlessFunctionTestTab.tsx` line 99
```suggestion
         <CodeEditor
            files={[
              {
                content: workflowRun.output,
                language: 'json',
                path: 'result.any',
              },
            ]}
            currentFilePath={'result.any'}
            options={{ readOnly: true, domReadOnly: true }}
          />
```

File: packages/twenty-front/src/modules/ui/layout/show-page/components/ShowPageRightContainer.tsx [hunk 2]
```diff
@@ -137,6 +140,13 @@ export const ShowPageRightContainer = ({
     isWorkflowEnabled &&
     targetableObject.targetObjectNameSingular ===
       CoreObjectNameSingular.WorkflowVersion;
+  const isWorkflowRun =
+    isWorkflowEnabled &&
+    targetableObject.targetObjectNameSingular ===
+      CoreObjectNameSingular.WorkflowRun;
+
+  const hideTabsNotWorkflowRelated =
```
Comment (thomtrp) [inline packages/twenty-front/src/modules/ui/layout/show-page/components/ShowPageRightContainer.tsx:148]: `isWorkflowRelated` would more consistent with the actual value

**Review by thomtrp — APPROVED** [review]

File: packages/twenty-front/src/modules/ui/layout/show-page/components/ShowPageRightContainer.tsx (hunk 2 above)
Comment (Devessier) [reply packages/twenty-front/src/modules/ui/layout/show-page/components/ShowPageRightContainer.tsx:148]: Done

File: packages/twenty-front/src/modules/workflow/components/WorkflowRunOutputVisualizer.tsx (hunk 1 above)
Comment (Devessier) [reply packages/twenty-front/src/modules/workflow/components/WorkflowRunOutputVisualizer.tsx]: Great idea! I made the change. It looks like that, now:

CleanShot 2024-10-17 at 14 00 15@2x

**Review by martmull — COMMENTED** [review]
One required change then LGTM

File: packages/twenty-front/src/modules/settings/serverless-functions/components/tabs/SettingsServerlessFunctionTestTab.tsx [hunk 3]
```diff
@@ -78,37 +79,31 @@ export const SettingsServerlessFunctionTestTab = ({
...
-            height={200}
-            onChange={setSettingsServerlessFunctionInput}
-          />
+          <SettingsServerlessFunctionCodeEditorContainer>
+            <CodeEditor
+              value={settingsServerlessFunctionInput}
+              language="json"
+              height={200}
+              onChange={setSettingsServerlessFunctionInput}
+              options={{ readOnly: true, domReadOnly: true }}
```
[the remark this answers is not in the document]
Comment (Devessier) [reply packages/twenty-front/src/modules/settings/serverless-functions/components/tabs/SettingsServerlessFunctionTestTab.tsx]: Thanks for catching this. It's fixed.

**Review by martmull — APPROVED** [review]
LGTM

**Pull request merged**